packages/desktop-client/src/components/util/GenericInput.js

```diff
@@ -45,23 +45,21 @@ export default function GenericInput({
     case 'id':
       switch (field) {
         case 'payee':
-          if (payees.length > 0) {
```

Comment:
It did not show the payee field before if there were no payees. So you just get.. nothing. Which is not great UX.

Now we will show an empty payee field. And if you start typing in it - you will get the "create payee" option. Which is slightly better.